### ISSUE TITLE
Can't launch emerald wallet

### ISSUE THREAD

**Issue by phyro** [ETCDEVTeam/emerald#4]
```
> emerald wallet
Couldn't get a file descriptor referring to the console
exec: Couldn't get a file descriptor referring to the console
```
mac hardcoded here might be causing this https://github.com/ETCDEVTeam/emerald/blob/master/index.js#L19

Thanks!

**Comment by shanejonas** [comment]
i'll be workin on getting the x-platform wallet builds going. should make the installation faster too

**Comment by shanejonas** [comment]
fixed in #7
